en/docs/SetupAndInstall/DeployingWSO2APIManager/ThirdPartyKeyManager/configuring-wso2-identity-server-as-a-key-manager.md

```diff
@@ -138,9 +138,9 @@ The following instructions are only applicable in the following scenarios:
                 <ServerURL>https://${gateway-server-host}:{port}/services/</ServerURL>
         ```
 
-        -   If you are working with a **hybrid single node or active-active node** deployment where WSO2 IS is the Key Manager and the rest of the API-M components are in one node, you need to replace `               {gateway-server-host}              ` with the host of the **WSO2** **API-M** node.
-        -   If you are working with a **single** **Gateway** in distributed set up, you need to replace `               {gateway-server-host}              ` with the host of the **Gateway** node.
-        -   If you are working with **Gateways** in a **High Availability (HA)** set up that uses a **shared file system** (e.g., NFS), you need to replace `               {gateway-server-host}              ` with the host of **any** **Gateway** node as all nodes have the worker manager capability when using shared file system.
+        -   If you are working with a **hybrid single node or active-active node** deployment where WSO2 IS is the Key Manager and the rest of the API-M components are in one node, you need to replace `               {gateway-server-host}              ` with the host of the **WSO2API-M** node.
+        -   If you are working with a **singleGateway** in distributed set up, you need to replace `               {gateway-server-host}              ` with the host of the **Gateway** node.
```

Comment:
Space is missing.

Reply:
done

Reply:
Changes in the configuring-wso2-identity-server-as-a-key-manager.md file are not available in the commit on Sep 9, 2019 - https://github.com/wso2/docs-apim/pull/19/commits/d2c3ff8450dd3cdcef80a86027b482e4ec4f5410

Reply:
The fix is available in the commit on Sep 10, 2019 - https://github.com/wso2/docs-apim/pull/19/commits/8866914ed8e436a24bae938b18fe042f0322edab